### DISCUSSION TITLE
Asking opinions on: My usage of issues

### DISCUSSION THREAD

**Discussion by dreamy32** [spaciousejar/learn-git-with-me#14] [General]
Hello everyone,

I’m reaching out to get your thoughts on how I’m using issues on this public repo.

Up until now, I’ve mostly used issues for my private projects, so I want to make sure I’m on the right track here!

Looking forward to contributing and hearing your feedback.

Cheers,
DrSeek

**Comment by spaciousejar** [comment]
Hi @dreamy32,

It’s fantastic that you’re seeking feedback on using issues in the public repository. 

You are good at using issues ✨

few tips to consider:

1. **Clarity**: Ensure each issue has a clear title and description so others can easily understand the context.

2. **Labels**: Use labels to categorize issues (e.g., bug, enhancement, question). This will help you organize and prioritize tasks effectively.

3. **Assigning**: If possible, assign issues to team members to clarify ownership and accountability.

If you have specific questions, feel free to ask. I look forward to your contributions!

**Reply by dreamy32** [reply]
Thanks for your feedback Spaciouse. Regarding labels and assigning people, I do not have the proper rights to do so, I can only create issues. I'd be glad to do so!
